pkg/restapi/webhook/webhook.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package webhook
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/hyperledger/aries-framework-go/pkg/common/log"
+)
+
+var logger = log.New("aries-framework/agentd")
+
+type dispatcher struct {
+	client         *http.Client
+	subscriberURLs []string
+	mu             *sync.RWMutex
```

Comment:
rwmutex pointers are only to share the mutex between goroutines/functions, so no need of pointer here.

use `mu             sync.RWMutex`

Reply:
Done!